src/RandomColor.js

```diff
@@ -0,0 +1,8 @@
+export function getRandomColor () {
```

Comment:
This is the coolest thing. You came up with a random color generator! The ability to come up with an idea outside of what you've seen before and try it out is the essence of programming.